packages/web/src/pages/settings-page/components/desktop/ManagerMode/RemoveManagerConfirmationContent.tsx

```diff
@@ -0,0 +1,90 @@
+import { useCallback, useEffect, useState } from 'react'
+
+import { useRemoveManager } from '@audius/common/api'
+import { Status } from '@audius/common/models'
+import { Button, Flex, Text } from '@audius/harmony'
+
+const messages = {
+  removeManagerConfirmation:
+    'Are you sure you want to remove this manager from your account?',
+  stopManagingConfirmation:
+    'Are you sure you want to stop managing this account?',
+  cancel: 'Cancel',
+  remove: 'Remove',
+  errorGeneral: 'Something went wrong.'
+}
+
+type RemoveManagerConfirmationContentProps = {
+  isStopManaging?: boolean
+  onSuccess: () => void
+  onCancel: () => void
+  userId: number
+  managerUserId: number
+}
+
+export const RemoveManagerConfirmationContent = ({
+  isStopManaging,
+  userId,
+  managerUserId,
+  onSuccess,
+  onCancel
+}: RemoveManagerConfirmationContentProps) => {
+  const [removeManager, result] = useRemoveManager()
+  const [error, setError] = useState<string | null>(null)
+  const { status } = result
+
+  const handleDelete = useCallback(() => {
+    if (!userId || !managerUserId) return
+    removeManager({ userId, managerUserId })
+  }, [userId, managerUserId, removeManager])
+
+  useEffect(() => {
+    if (status === Status.SUCCESS) {
+      onSuccess()
+    }
+  }, [status, onSuccess])
+
+  useEffect(() => {
+    if (status === Status.ERROR) {
+      setError(messages.errorGeneral)
```

Comment:
Does error need to be its own state? Can't we derive it from status each render like we do with is submitting?

Reply:
No it doesn't, will clean up